### PR TITLE
Connection of TechNotify project with Mongo db.

### PR DESCRIPTION
**Pull request by Arundhathy-k**
Connected TechNotify Application with Mongo db to perform database operations.

### CODE REVIEW

File: mongo-dao/pom.xml [hunk 1]
```diff
@@ -10,11 +10,28 @@
...
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
-
+    <dependencies>
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-data-mongodb</artifactId>
+    </dependency>
+    <dependency>
```
Comment (dhanyesh) [inline mongo-dao/pom.xml:25]: remove this dependency

File: mongo-dao/src/main/java/com/kovan/repository/TestRepository.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.kovan.repository;
+
+import com.kovan.entity.TestEntity;
+import org.springframework.data.mongodb.repository.MongoRepository;
+
```
Comment (dhanyesh) [inline mongo-dao/src/main/java/com/kovan/repository/TestRepository.java:6]: Put @Repository annotation on top of it

File: newsService/src/main/java/com/kovan/api/model/TestRequest.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package com.kovan.api.model;
+
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+
+@Data
+@Document(collection = "TestData")
```
Comment (dhanyesh) [inline newsService/src/main/java/com/kovan/api/model/TestRequest.java]: why do we need this ?

File: newsService/src/main/java/com/kovan/app/service/TestService.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+public class TestService {
+
+
+    @Autowired
+    private TestRepositoryService service;
+
+
+    public TestDto addData(TestRequest testRequest) {
+
+        if (testRequest == null) {
```
Comment (dhanyesh) [inline newsService/src/main/java/com/kovan/app/service/TestService.java]: Don't check null like this use StringUtils

File: mongo-dao/src/main/java/com/kovan/repository/TestRepository.java (hunk 2 above)
Comment (Arundhathy-k) [reply mongo-dao/src/main/java/com/kovan/repository/TestRepository.java:6]: Added @Repository annotation.

File: newsService/src/main/java/com/kovan/api/model/TestRequest.java (hunk 3 above)
Comment (Arundhathy-k) [reply newsService/src/main/java/com/kovan/api/model/TestRequest.java]: Removed @Document(collection = "TestData") from TestRequest class.

File: newsService/src/main/java/com/kovan/app/service/TestService.java (hunk 4 above)
Comment (Arundhathy-k) [reply newsService/src/main/java/com/kovan/app/service/TestService.java]: Issue resolved.

File: mongo-dao/pom.xml (hunk 1 above)
Comment (Arundhathy-k) [reply mongo-dao/pom.xml:25]: Dependency removed.

**Pull request merged**